cmd/service-controller/console_server.go

```diff
@@ -417,6 +417,7 @@ func (server *ConsoleServer) listen() {
 	r.Handle("/policy/service/{name}", authenticated(server.policies.service()))
 	r.Handle("/policy/incominglink", authenticated(server.policies.incomingLink()))
 	r.Handle("/policy/outgoinglink/{hostname}", authenticated(server.policies.outgoingLink()))
+	r.Handle("/policy/dump", authenticated(server.policies.dump()))
```

Comment:
Again, 'dump' to me feels a bit odd in the resource path. Maybe /policy/all? Again though, this is a minor personal nit pick, not necessarily an issue.

Reply:
Policy all or policy list might be more meaningful. With that, I also need to change the
command to `get policies list` (or all). 
@grs @ajssmith @nluaces which one sounds better?